### PR TITLE
Image scale

### PR DESCRIPTION
**Pull request by akaDuality**
Managed image scale: AppKit knows about image's scale, but UIKit has no idea about that. As a result we should manage it by ourselves to keep controls aligned to image.

Also slightly prepares for feature `frames` of #207 when several images will be placed on board like figma. For know it allows to keep read-write code at single place for AppKit and UIKit.

- [x] Wrap files in `Frame` folder for feature use
- [x] Backward support for old files without `Frame` folder
- [x] Add tests
   - [x] file reading and backward compatibility
   - [x] scale management
- [x] Fix thumbnailing for new file format

### CODE REVIEW

File: VoiceOver Designer.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -603,7 +603,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				COMBINE_HIDPI_IMAGES = YES;
 				CURRENT_PROJECT_VERSION = 3;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = P67Q4Q7HA9;
```
[the remark this answers is not in the document]
Comment (drucelweisse) [reply VoiceOver Designer.xcodeproj/project.pbxproj:606]:
```suggestion
				DEVELOPMENT_TEAM = "";
```

File: VoiceOver Designer.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -659,7 +659,7 @@
 				CODE_SIGN_ENTITLEMENTS = MacOSPreview/MacOSPreview.entitlements;
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 3;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = P67Q4Q7HA9;
```
Comment (drucelweisse) [inline VoiceOver Designer.xcodeproj/project.pbxproj:662]:
```suggestion
				DEVELOPMENT_TEAM = "";
```

File: VoiceOver Designer.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -686,7 +686,7 @@
...
```
Comment (drucelweisse) [inline VoiceOver Designer.xcodeproj/project.pbxproj:689]:
```suggestion
				DEVELOPMENT_TEAM = "";
```

**Review by drucelweisse — CHANGES_REQUESTED** [review]

**Comment by akaDuality** [comment]
Have removed, thx!

**Review by drucelweisse — APPROVED** [review]

**Pull request merged**